src/components/data_display/helpers.ts

```diff
@@ -54,7 +55,11 @@ export const buildAllColumnDefinitions = async ({
   // Look up allowed Data views
   const dataTables = (await DBConnect.getAllTableNames()).map((tableName) => camelCase(tableName))
 
-  if (!dataTables.includes(tableName)) throw new Error(`Invalid table name: ${tableName}`)
+  const matchingTableName = dataTables.find(
+    (table) => table === dataTablePrefix + capitaliseFirstLetter(tableName) || table === tableName
+  )
+
+  if (!matchingTableName) throw new Error(`Invalid table name: ${tableName}`)
```

Comment:
Checks for *either* with or without prefix. We now pass along the matched table name rather than the requested table name, so whichever one was found (prefixed or non-prefixed) will continue to be used for remaining computation.

This means that "user" and "organisation" tables will continue to work as expected.